### PR TITLE
chore: cleanup and code inspection-based fixes

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by stefreak — APPROVED** [review]
Awesome! This looks good to me. There is a failing test though (I'm not sure if it's a flake).

@vvagaytsev How did you find the typos and the unused code? If you used a tool for that, we should add it to the CI / our linter configuration as well.

File: core/src/commands/get/get-eysi.ts [hunk 1]
```diff
@@ -21,8 +21,6 @@ export class GetEysiCommand extends Command {
     Just try it.
   `
 
-  loggerType: "basic"
```
Comment (TimBeyer) [inline core/src/commands/get/get-eysi.ts:24]: I believe these are actually bugs where we needed to use `=` but accidentally used `:`

**Comment by vvagaytsev** [comment]
@stefreak I used the code inspection tool in WebStorm. There should be some tools that we can embed into the CI pipeline.

File: core/src/commands/get/get-eysi.ts (hunk 1 above)
Comment (vvagaytsev) [reply core/src/commands/get/get-eysi.ts:24]: Yes, looks like `=` should be used. But this field is no longer used in the code. So I removed it :)

**Comment by stefreak** [comment]
@vvagaytsev  I think for CI it would be paid: https://www.jetbrains.com/qodana/buy/?billing=yearly

**Pull request merged**